pkg/rkt-stage1/0011-Patch-to-mount-vol-stage2.patch

```diff
@@ -0,0 +1,122 @@
+diff --git a/build.sh b/build.sh
+index 04fceb6..9c623f3 100755
+--- a/build.sh
++++ b/build.sh
+@@ -24,7 +24,7 @@ then
+     exit 1
+ fi
+
+-execs="enter run stop udhcpc_script.sh launcher.sh"
++execs="enter run stop udhcpc_script.sh launcher.sh mount_disk.sh"
+ netplugins="main/ptp main/bridge main/macvlan main/ipvlan ipam/host-local meta/flannel meta/tuning"
+
+ # Clean the repo, but save the vendor area
+diff --git a/files/mount_disk.sh b/files/mount_disk.sh
+new file mode 100644
+index 0000000..d51f1e8
+--- /dev/null
++++ b/files/mount_disk.sh
+@@ -0,0 +1,62 @@
++#!/bin/sh
++
++mountPointLineNo=1
++ls /sys/block/ | grep xvd | while read -r disk ; do
++  echo "Processing $disk"
++
++  #Fetching Major and Minor device
++  IN=$(cat /sys/block/$disk/dev | tr ":" "\n")
++  major=$(echo ${IN} | cut -d' ' -f1)
++  minor=$(echo ${IN} | cut -d' ' -f2)
++
++  #Creating a block device under /dev with Major and minor devices
++  echo "Creating device file /dev/$disk"
++  mknod /dev/$disk b $major $minor && \
++  echo "Successfully created device file for /dev/$disk" || \
++  echo "Failed to create device file for /dev/$disk"
++  echo
++
++  #Creating partition on block device
```

Comment:
Why are you bothering with creating partitions? Why not use the disk as-is @adarsh-zededa ?

Reply:
Went ahead with the idea that the scratch image when mounted will not have any (primary) partition. Noticed there was already a primary partition created, hence no need to create it manually. Removed this section of code.